src/cls/otp/cls_otp_ops.h

```diff
@@ -0,0 +1,150 @@
+#ifndef CEPH_CLS_OTP_OPS_H
+#define CEPH_CLS_OTP_OPS_H
+
+#include "include/types.h"
+#include "include/utime.h"
+#include "cls/otp/cls_otp_types.h"
+
+struct cls_otp_set_otp_op
+{
+  list<rados::cls::otp::otp_info_t> entries;
+
+  cls_otp_set_otp_op() = default;
```

Comment:
the compiler will generate this for you "if no other constructor is explicitly declared" - https://en.wikipedia.org/wiki/Special_member_functions